### DISCUSSION TITLE
Jellyfin update 10.10.0

### DISCUSSION THREAD

**Discussion by gladiatormax** [tteck/Proxmox#3984] [Help Needed] [answered]
After update to version 10.10.0 the server version stays on 10.9.11 ?
Did the Update with the update command.
![image](https://github.com/user-attachments/assets/3aef670d-081a-4334-8f6d-560ad5d406c9)

**Comment by Rockhound53** [comment]
I type the update Command an can upgrade the Server Version, but the Web an Build stays at 10.9.11?

![image](https://github.com/user-attachments/assets/4a6921a5-16ff-4a94-b707-9fd81c08a44e)

**Reply by gladiatormax** [reply]
Its getting stranger, is there a log off the update command somewhere?

**Reply by tteck** [reply]
The update is using Jellyfin's apt repository.
https://github.com/tteck/Proxmox/blob/06742f7ef5773b82a4b3cd127a4fdae16d058251/install/jellyfin-install.sh#L40-L50

**Reply by gladiatormax** [reply]
I did an update manually with the command from Jellyfin → curl https://repo.jellyfin.org/install-debuntu.sh | sudo bash and now its on 10.10.0.

**Comment by Rockhound53** [comment]
I restore my Backup an this worked for me:

curl https://repo.jellyfin.org/install-debuntu.sh | sudo bash 

than the version was 10.9.11

then type update in the shell an now are all Versions 10.0.10...

**Comment by makstech** [comment, marked as answer]
The issue is that the new `jellyfin` and/or `jellyfin-server` packages rely on `jellyfin-ffmpeg7` instead of `jellyfin-ffmpeg6`.  

To fix the issue, connect to LXC and run 
```bash
apt-get update && apt-get -y upgrade jellyfin-server
```
